### DISCUSSION TITLE
Synthesis完I/O爆掉

### DISCUSSION THREAD

**Discussion by v00000v** [bol-edu/HLS-SOC-Discussions#109] [Q&A]
請問在synthesis之後I/O大幅超出上限時有什麼解決方法嗎，謝謝!
我選的板子是xc7z020clg400-1
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/123727462/86e49d1a-6ed4-40b5-a45f-e168731a8894)

**Comment by ZheChen-Bill** [comment]
因為我們的設計有許多的I/O pin，因此在合成時會看到I/O的使用量遠大於可以使用的I/O，也因此在這次實驗只需要進行到synthesis而不用進行implementation。
